api-service/service/oracle_database_license_types_test.go

```diff
@@ -226,9 +227,9 @@ func TestGetLicensesCompliance(t *testing.T) {
 	require.NoError(t, err)
 
 	expected := []dto.LicenseCompliance{
-		{LicenseTypeID: "PID001", ItemDescription: "itemDesc1", Metric: "Processor Perpetual", Consumed: 10, Covered: 7, Compliance: 0.7, Unlimited: false},
+		{LicenseTypeID: "PID001", ItemDescription: "itemDesc1", Metric: "Processor Perpetual", Consumed: 11, Covered: 7, Compliance: 7.0 / 11.0, Unlimited: false},
 		{LicenseTypeID: "PID002", ItemDescription: "itemDesc2", Metric: "Named User Plus Perpetual", Consumed: 475, Covered: 75, Compliance: 75.0 / 475.0, Unlimited: false},
-		{LicenseTypeID: "PID003", ItemDescription: "itemDesc3", Metric: "Computer Perpetual", Consumed: 11, Covered: 0.5, Compliance: 1, Unlimited: true},
+		{LicenseTypeID: "PID003", ItemDescription: "itemDesc3", Metric: "Computer Perpetual", Consumed: 10, Covered: math.RoundToEven(0.5), Compliance: 1, Unlimited: true},
```

Comment:
Don't use the math function here, put directly the value

Reply:
If it was a real number, it would be ok to put a calculation like `75.0 / 475.0`, but round shouldn't return an integer?...